scripts/exglobal_atm_analysis_finalize.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# exgdas_global_atm_analysis_finalize.py
+# This script creates an AtmAnalysis class
+# and runs the finalize method
+# which perform post-processing and clean up activities
+# for a global atm variational analysis
+import os
+
+from pygw.logger import Logger, logit
```

Comment:
```suggestion
from pygw.logger import Logger
```
`logit` is not used here.

Reply:
Change committed at c27618bb